pydeequ/configs.py

```diff
@@ -23,7 +23,9 @@ def _extract_major_minor_versions(full_version: str):
 @lru_cache(maxsize=None)
 def _get_spark_version() -> str:
     try:
-        spark_version = os.environ["SPARK_VERSION"]
+        spark_version = os.getenv("SPARK_VERSION")
+        if not spark_version:
+            spark_version = str(pyspark.__version__)
```

Comment:
I think technically pyspark version does not necessarily imply actual Spark version (most of times, they do match). Could we add an informational log message after this line like `f"SPARK_VERSION environment variable is not set, using Spark version from PySpark {spark_version} for Deequ Maven jars"`

Reply:
done